documentation/src/docs/controller-service.md

```diff
@@ -897,4 +897,4 @@ Resources<a name="resources"></a>
 ---------
 - [Pravega](http://pravega.io/)
 - [Code](https://github.com/pravega/pravega/tree/master/controller)
-- [Other Documents](http://pravega.io/docs/latest/)
+- [Other Documents](.)
```

Comment:
I don't think this change is correct, check the view of this file.

Reply:
I think this is fine, as this points to all the other documents available in Github.